### ISSUE TITLE
Make the official OWL RL tests part of the project

### ISSUE THREAD

**Issue by wrobell** [RDFLib/OWL-RL#14]
The readme file states this version passes the full batch of official OWL Full/RL tests. It would be good to make the tests part of the project to make collaboration easier and more constructive.

**Comment by blokhin** [comment]
Artur, I believe that's this: https://www.w3.org/2007/OWL/wiki/Full (namely, https://www.w3.org/2007/OWL/wiki/Test_Suite_Status). However the overall state is quite sad (404 pages, not maintained etc.)

**Comment by wrobell** [comment]
It seems like `cade2011-schneidsut-owlfullatp/testdata/coverage/testsuite-owl2-rdfbased.zip` contained in the above file is our starting point?

I do not see any license associated with the file, so I wonder if we can include it as part of the project?

**Comment by wrobell** [comment]
Also http://owl.semanticweb.org/exports/approved/index.html

**Comment by nicholascar** [comment]
I agree (as I said in your PR now approved) that we really do need the official test to actually know how the tool is performing in a comprehensive manner. I hope we can all start to outline the full set of tests first and then start chipping away at implementing them all soon. I may have more time to contribute over summer (Dec - Feb, here in Australia!),

**Comment by blokhin** [comment]
A series of the OWL Reasoner Evaluation workshops should be probably also mentioned here, namely:

* [OWL Reasoner Evaluation 2015](http://dl.kr.org/ore2015) ([report](https://doi.org/10.1007/s10817-017-9406-8) and [code](https://github.com/andreas-steigmiller/ore-competition-framework))
* [OWL Reasoner Evaluation 2016](https://www.w3.org/community/owled/workshop-2016) ([report](https://doi.org/10.1007/978-3-319-54627-8))
* 2017 ... 2019 (?)
